classes/v1migration/v1migration.php

```diff
@@ -550,7 +550,7 @@ private function log_success_migration_event($turnitintooltwoid, $course_id, $v1
     }
 
     /**
-     * Get assignments for migrated data table. Called from ajax.php via turnitintooltwo_extra-2024020601.min.js.
+     * Get assignments for migrated data table. Called from ajax.php via turnitintooltwo_extra-2024032801.min.js.
```

Comment:
noticed that the version here says `2024032801`, but saw `2024032901` above. I think this should be `2024032901` so that it aligns with the date it is actually releasing on